pom.xml

```diff
@@ -14,18 +14,80 @@
     <artifactId>container-client</artifactId>
     <version>2.1.0-SNAPSHOT</version>
 
+    <repositories>
+        <repository>
```

Comment:
do you really need to add Maven central?

Reply:
fixed